src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -813,6 +813,7 @@ PooledHashSet<CaptureId> adjustAndGetEntryState(ArrayBuilder<PooledHashSet<Captu
                                 ((isFirstOperandOfDynamicOrUserDefinedLogicalOperator(reference) ||
                                      isIncrementedNullableForToLoopControlVariable(reference) ||
                                      isConditionalAccessReceiver(reference) ||
+                                     isCoalesceAssignment(reference) ||
```

Comment:
>isCoalesceAssignment [](start = 37, length = 20)

Could we tighten this exception to only be for null-coalescing assignments where the LHS is a property?

Reply:
I've updated the syntax used in the CFG to be a little more precise and removed this entirely.

---
In reply to: [293124296](https://github.com/dotnet/roslyn/pull/36329#discussion_r293124296) [](ancestors = 293124296)

Reply:
To be clear, I actually didn't need it last iteration, I had left it in but had apparently already removed the need.

---
In reply to: [294962493](https://github.com/dotnet/roslyn/pull/36329#discussion_r294962493) [](ancestors = 294962493,293124296)